GoogleMapsComponents/wwwroot/objectManager.js

```diff
@@ -160,11 +160,17 @@ window.googleMapsObjectManager = {
             });
     
             //Wait for promise
-            let result = await promise;
-            obj.setDirections(result);
-            
-            let jsonRest = JSON.stringify(cleanDirectionResult(result));
-            return jsonRest;
+            try {
+                let result = await promise;
+                obj.setDirections(result);
+                
+                let jsonRest = JSON.stringify(cleanDirectionResult(result));
+                return jsonRest;
+            } catch (error) {
+                console.log(error);
+                return error;
+            }
```

Comment:
So you handling exception in two places. Since you put error to console here why do same in class.
Also error could be anything, not promise error.
Also promise error should be catch like (i could we very wrong. js is nightmare for me) this
promise.catch(error => { 
    console.log(error);
});

Reply:
Because Blazor is waiting for a response... If I don't respond back, it will throw an exception if nothing comes back.